code/style.css

```diff
@@ -1,7 +1,431 @@
+  font-family: "permanent marker";
+  font-size: 34px;
+  color: #FFF7AE;
+  text-shadow: 2px 2px #9D98FA;
+}
+
+/* NAVBAR */
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  width: 100vw;
+  margin: auto;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 30px;
+}
+
+ul {
+  width: 100vw;
+  display: flex;
+  list-style-type: none;
+  justify-content: center;
+  padding-inline-start: 0;
+}
+
+li {
+  padding: 3%;
+  justify-content: center;
+}
+
+li a {
+  color: #FFF7AE;
+  font-size: 18px;
+  font-weight: 600;
+  text-decoration: none;
+}
+
+li a:hover {
+  text-decoration: underline;
+}
+
+h2 {
+  font-size: 16px;
+  margin-bottom: 8%;
+}
+
+
+/* MAIN CONTENT */
+
+.flexParent {
+  display: flex;
+  flex-direction: column;
+}
+
+
+/* FORMS CONTAINER */
+
+.formWrapper {
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: auto;
+}
+
+#form {
+  margin-bottom: 30px;
+  padding: 10px 40px 10px 40px;
+  font-size: 12px;
+  background-color: #E0D4FE;
+  opacity: 0.95;
+  position: relative;
+  z-index: 20;
+  border-radius: 2%;
+}
+
+label {
+  margin-bottom: 2%;
+  margin-top: 5%;
+}
+
+.blockLabel, .blockInput {
+  display: block;
+  
+}
+
+.radioGender {
+  display: inline-block;
+  padding: 2%;
+  margin-top: 4%;
+  margin-bottom: 2%;
+}
+
+#termsConditions, #newsletter {
+  padding: 2%;
+}
+
+.submitButton {
+  background-color: #FFF7AE;
+  padding: 5%;
+  margin-top: 5%;
+  margin-bottom: 5%;
+  border: solid;
+  border-width: 1px;
+  border-radius: 8%;
+  border-color: #9D98FA;
+  
+}
+
+/* ACCORDION CONTAINER */
+
+.accordionWrapper {
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  justify-content: center;
+}
+
+#accordion {
+  font-size: 12px;
+  margin-bottom: 10px;
+  padding: 10px 40px 30px 40px;
+  background-color: #E0D4FE;
+  opacity: 0.95;
+  position: relative;
+  z-index: 20;
+  border-radius: 5px;
+}
+
+.question {
+  background-color: #FFF7AE;
+  padding: 8%;
+  border:solid;
+  border-width: 1px;
+  border-color: #fcf08b;
+  font-weight: 600;
+}
+
+.answer {
+  line-height: 1.5;
+  background-color: #fffbdb;
+  border:solid;
+  border-width: 1px;
+  border-color: #fcf08b;
+  padding: 8%;
+}
+
+#question1, #question2, #question3, #question4 {
+  margin: 0;
+  min-width: 100%;
+}
+
+
+/* FOOTER */
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 20px;
+  padding: 5%;
+  position: relative;
+  z-index: 20;
+}
+
+.footerText {
+  display: flex;
+  font-family: "Permanent Marker";
+  font-size: 14px;
+  color: #FFF7AE;
+  margin-bottom: 8%;
+}
+
+.footerCred {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  margin: 2px;
+  color: #FFF7AE;
+}
+
+
+/* --------!!! MEDIA QUERIES !!!-------------- */
+
+/* ------------- TABLET SIZE ----------- */
+
+@media (min-width: 667px) and (max-width: 1024px)  {
```

Comment:
The appearance of the tablet version was the nicest one in my opinion 💯

Reply:
I agree 100%, thanks for noticing! 😄